src/status_im/contexts/wallet/common/collectibles_tab/view.cljs

```diff
@@ -4,11 +4,28 @@
     [quo.theme]
     [react-native.core :as rn]
     [status-im.common.resources :as resources]
+    [status-im.contexts.wallet.collectible.utils :as utils]
     [status-im.contexts.wallet.common.empty-tab.view :as empty-tab]
     [utils.i18n :as i18n]))
 
+(defn- render-fn
+  [{:keys [preview-url collection-data ownership collectible-data] :as collectible} index address
+   on-press]
+  (let [total-owned (utils/calculate-owned-collectible ownership address)]
+    [quo/collectible-list-item
+     {:type                 :card
+      :image-src            (:uri preview-url)
+      :avatar-image-src     (:image-url collection-data)
+      :collectible-name     (:name collection-data)
+      :collectible-mime     (:animation-media-type collectible-data)
+      :gradient-color-index (keyword (str "gradient-" (inc (mod index 5))))
+      :counter              (utils/collectible-owned-counter total-owned)
+      :container-style      {:padding 8}
+      :on-press             #(when on-press
+                               (on-press collectible))}]))
+
 (defn- view-internal
-  [{:keys [theme collectibles filtered? on-collectible-press on-end-reached]}]
+  [{:keys [theme collectibles filtered? on-collectible-press on-end-reached current-account-address]}]
```

Comment:
I believe we should get current-account-address from a sub instead. 
wdyt?

Reply:
In its parent, it's coming from a sub. The same approach is true for `collectibles`. I wanted to keep the same logic for them